test/test_generate_yaml.py

```diff
@@ -10,7 +10,8 @@ def test_yaml_gen_yaml(parm_dir):
     output_file = os.path.join(os.getcwd(), 'testoutput', 'test_yaml_gen.yaml')
     template = os.path.join(parm_dir, 'atm/variational/3dvar_dripcg.yaml')
     config_dict = {
-        'paths': f'$<< {parm_dir}/atm/common/paths.yaml',
+        'BERROR_YAML': '${PARMgfs}/atm/berror/staticb_bump.yaml',
```

Comment:
If users want to use `staticb_identity.yaml`, is this the line they need to change?

Reply:
Correct.

Reply:
This is for the unit test. Most users would change it in `sample.yaml`.